tests/unit/amo/components/TestSearchContextCard.js

```diff
@@ -0,0 +1,158 @@
+import { shallow } from 'enzyme';
+import React from 'react';
+
+import {
+  SearchContextCardBase,
+  mapStateToProps,
+} from 'amo/components/SearchContextCard';
+import { searchLoad, searchStart } from 'core/actions/search';
+import { dispatchClientMetadata, fakeAddon } from 'tests/unit/amo/helpers';
+import { getFakeI18nInst } from 'tests/unit/helpers';
+
+
+describe('SearchContextCard', () => {
+  function render(props = {}) {
+    let { store } = props;
+
+    if (!store) {
+      store = dispatchClientMetadata().store;
+      store.dispatch(searchStart({ filters: { query: 'test' } }));
+    }
+
+    return shallow(
+      <SearchContextCardBase
+        {...mapStateToProps(store.getState())}
```

Comment:
I think that's a great approach. 👍

I'd be happy to go through all the components that need it and do it in one PR rather than in this one (it's already pretty big). But let me know if you'd rather me do it here.